src/components/VideoPlayer/VideoPlayer.jsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable */
+import React, { Fragment, useState, useContext, useEffect } from 'react';
+import { Button } from 'semantic-ui-react';
+import { useAuth } from '../../providers/Auth/Auth.provider';
+import VideosContext from '../../context/VideosContext';
+
+import { FAVORITES_LIST, FAVORITES_ID } from '../../utils/constants';
+import { storage } from '../../utils/storage';
+import { getIdx } from '../../utils/utils';
+
+import moment from 'moment';
+const VideoPlayer = () => {
+  let favoritesList = storage.get(FAVORITES_LIST);
+  let favoritesId = storage.get(FAVORITES_ID);
+  const { authenticated } = useAuth();
+  const { selectedVideo, id } = useContext(VideosContext);
+  const [isFav, setIsFav] = useState(favoritesList[`${id}`]);
+
+  useEffect(() => {
+    setIsFav(favoritesId[`${id}`]);
+  }, [selectedVideo]);
+
+  const handleSetFavorites = () => {
+    if (favoritesId[`${id}`]) {
+      const i = getIdx(favoritesList, id);
+      const newList = [...favoritesList.slice(0, i), ...favoritesList.slice(i + 1)];
+      favoritesList = newList;
+      delete favoritesId[`${id}`];
+    } else {
+      favoritesId[`${id}`] = true;
+      favoritesList.push(selectedVideo);
+    }
+    storage.set(FAVORITES_LIST, favoritesList);
+    storage.set(FAVORITES_ID, favoritesId);
+    setIsFav(favoritesId[`${id}`]);
+  };
+  return (
+    <Fragment>
```

Comment:
Nitpick 
You can use just `<></>` instead of Fragment :)

Reply:
Nice trick!